monome/grid.go

```diff
@@ -24,9 +24,21 @@ type KeyEvent struct {
 	S int32
 }
 
+type TiltEvent struct {
```

Comment:
The `TiltEvent` type definition should go after the `KeyEvent` methods, so that every type is immediately followed by its methods.